### DISCUSSION TITLE
Getting latex to render in actual book

### DISCUSSION THREAD

**Discussion by loftusa** [executablebooks/meta#276] [Q&A]
hey all, just started using jupyter books!

I can't get my latex to render in the actual book [here](http://docs.neurodata.io/graph-stats-book/representations/ch6/joint-representation-learning.html). It renders fine in a jupyter notebook. I enabled [amsmath](https://github.com/neurodata/graph-stats-book/blob/master/network_machine_learning_in_python/_config.yml#L16) in my `_config.yml` file. Is there just something I'm missing?

Thanks!

**Comment by AakashGfude** [comment]
Hey, @loftusa can you send us the error you encountered?

Also, the present latex infrastructure is a bit lacking in `jupyter-book`. Which we are trying to fix using this PR https://github.com/executablebooks/jupyter-book/pull/1167 , by making `jupyterbook-latex` as a default package.
We have delegated the task of handling the latex side of things to `jupyterbook-latex`. 

For now, what you can do is, add `jupyterbook-latex` explicitly to your extension list and check for us if it fixes things. If not, create an issue or two in https://github.com/executablebooks/jupyterbook-latex  😬 .

**Reply by loftusa** [reply]
Hey! I didn't actually encounter an error -- the [actions](https://github.com/neurodata/graph-stats-book/actions) are passing, just not rendering latex in the final product. I'll look into adding the extension.
